youtube_dl/extractor/markiza.py

```diff
@@ -87,37 +124,39 @@ class MarkizaPageIE(InfoExtractor):
             'skip_download': True,
         },
     }, {
-        'url': 'http://dajto.markiza.sk/filmy-a-serialy/1774695_frajeri-vo-vegas',
+        'url': 'https://dajto.markiza.sk/filmy-a-serialy/1774695_frajeri-vo-vegas',
         'only_matching': True,
     }, {
-        'url': 'http://superstar.markiza.sk/aktualne/1923870_to-je-ale-telo-spevacka-ukazala-sexy-postavicku-v-bikinach',
+        'url': 'https://superstar.markiza.sk/aktualne/\
+        1923870_to-je-ale-telo-spevacka-ukazala-sexy-postavicku-v-bikinach',
         'only_matching': True,
     }, {
-        'url': 'http://hybsa.markiza.sk/aktualne/1923790_uzasna-atmosfera-na-hybsa-v-poprade-superstaristi-si-prve-koncerty-pred-davom-ludi-poriadne-uzili',
+        'url': 'https://hybsa.markiza.sk/aktualne/1923790_uzasna-atmosfera-na-hybsa-v-poprade-superstaristi-si-prve-koncerty-pred-davom-ludi-poriadne-uzili',
         'only_matching': True,
     }, {
-        'url': 'http://doma.markiza.sk/filmy/1885250_moja-vysnivana-svadba',
+        'url': 'https://doma.markiza.sk/filmy/1885250_moja-vysnivana-svadba',
         'only_matching': True,
     }, {
-        'url': 'http://www.tvnoviny.sk/domace/1923887_po-smrti-manzela-ju-cakalo-poriadne-prekvapenie',
+        'url': 'https://www.tvnoviny.sk/domace/1923887_po-smrti-manzela-ju-cakalo-poriadne-prekvapenie',
         'only_matching': True,
     }]
 
     @classmethod
     def suitable(cls, url):
-        return False if MarkizaIE.suitable(url) else super(MarkizaPageIE, cls).suitable(url)
```

Comment:
The project doesn't enforce a line length and \-continuations are an accident waiting to happen. Consider using `()` if the line is really too long (eg Flake8 complains):
```
        return (
            False if MarkizaIE.suitable(url)
            else super(MarkizaPageIE, cls).suitable(url)
        )
```
Although I don't advise changing the line (a similar expression is used in quite a few other extractors), isn't `not MarkizaIE.suitable(url) and super(MarkizaPageIE, cls).suitable(url)` a shorter equivalent expression?